docs/user-guide/persistent-volumes/index.md

```diff
@@ -216,3 +229,106 @@ spec:
 ### A Note on Namespaces
 
 `PersistentVolumes` binds are exclusive, and since `PersistentVolumeClaims` are namespaced objects, mounting claims with "Many" modes (`ROX`, `RWX`) is only possible within one namespace.
+
+## StorageClasses
+
+Each `StorageClass` contains the fields `provisioner` and `parameters`, which 
+are used when a `PersistentVolume` belonging to a class needs to be dynamically
+ provisioned.
+
+The name of a `StorageClass` object is significant, and is how users can 
+request a particular class. Administrators set the name and other parameters 
+of a class, all of which are opaque to users, when first creating 
+`StorageClass` objects, and the objects cannot be updated once they are 
+created.
+
+```yaml
+kind: StorageClass
+apiVersion: extensions/v1beta1
+metadata:
+  name: standard
+provisioner: kubernetes.io/aws-ebs
+parameters:
+  type: gp2
+```
+
+### Provisioner
```

Comment:
OK, I may wait for resolution of some open questions in https://github.com/kubernetes/kubernetes/pull/30900